en/12_Project_Governance/05_Release_policy.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Release policy
+summary: Outline the release lifecycle and support commitment for Silverstripe CMS
+icon: code-branch
+---
+
+# Silverstripe CMS release policy
+
+This document outlines the unified release policy for Silverstripe CMS and its commercially supported modules. Its purpose is to provide clear guidance on the types of releases, their cadence, scope, and support lifecycle, ensuring predictability and stability for our users and developers.
+
+## Release types and cadence
+
+Silverstripe CMS adheres to [Semantic versioning](https://semver.org/) for its release philosophy.
+
+### Major releases `(x.0.0)`
+
+Major releases are the cornerstone for introducing significant evolution and breaking changes to the Silverstripe CMS platform.
+
+- **Purpose:** To deliver breaking changes and architectural updates that cannot be shipped in minor releases due to backward compatibility constraints.
+
+- **Backward compatibility:** Major releases are **not backward compatible**.
+
+- **Frequency:** New Silverstripe CMS major versions are released every **two years**.
+
+- **Tagging period:** New major releases are typically tagged as stable between **April and June** of their release year.
+
+- **Scope:** This policy applies to all Silverstripe CMS commercially supported modules. At launch, the list of commercially supported modules for that specific major release line will be explicitly defined.
+
+- **Pre-release period:** A stable major release is always preceded by a **beta period of at least three months** to allow for extensive testing and feedback.
+
+- **Dependency alignment:** At launch, Silverstripe CMS major releases aim to:
+
+  - Support all PHP versions currently in full support.
+  - Have all PHP and JavaScript dependencies on their latest stable versions.
+
+- **Upgrade path:** Silverstripe CMS commits to providing a clear upgrade path to new major releases, including comprehensive documentation on all breaking changes and migration guides.
```

Comment:
This seems to all be new?

Reply:
Correct

Reply:
........ that's gonna make this really hard to review but I'll try. I suspect a lot of my review comments will be effectively a rewrite that brings this more in-line with the combined language of the current policies.